storybook/decorators/with-max-width-wrapper.js

```diff
@@ -27,14 +17,19 @@ const Indicator = styled.div`
 `;
 
 export const WithMaxWidthWrapper = ( Story, context ) => {
-	if ( context.globals.maxWidthWrapper === 'none' ) {
+	/**
+	 * A Storybook decorator to wrap a story in a div applying a max width.
+	 * This can be used to simulate real world constraints on components
+	 * such as being located within the WordPress editor sidebars.
+	 */
+	const maxWidth = context.globals.maxWidthWrapper;
+	if ( ! context.globals.maxWidthWrapper ) {
```

Comment:
You could just use the constant we just created:

```suggestion
	if ( ! maxWidth ) {
```

Reply:
Of course. I missed that..